src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDropTestHelper.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Linq;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace UITests.Windows_UI_Xaml.DragAndDrop
+{
+	internal class DragDropTestHelper
+	{
+		private readonly TextBlock _output;
+
+		public DragDropTestHelper(TextBlock output)
+		{
+			_output = output;
+		}
+
+		public void SubscribeDragEvents(FrameworkElement elt)
+		{
+			elt.DragStarting += (snd, e) =>
+			{
+				switch (snd)
+				{
+					case Image img when img.Source is BitmapImage bmp:
+						e.Data.SetBitmap(RandomAccessStreamReference.CreateFromUri(bmp.UriSource));
+						break;
+
+					case TextBlock text:
+						e.Data.SetText(text.Text);
+						break;
+
+					case HyperlinkButton link:
+						e.Data.SetWebLink(link.NavigateUri);
+						break;
+				}
+
+				Log(snd, "STARTING", e.Data.GetView());
+			};
+
+			elt.DropCompleted += (snd, e)
+				=> _output.Text += $"[{GetName(snd)}] COMPLETED {e.DropResult}\r\n"; ;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)